alvr/server/build.rs

```diff
@@ -12,12 +12,34 @@ fn get_ffmpeg_path() -> PathBuf {
         .join("ffmpeg");
 
     if cfg!(target_os = "linux") {
-        ffmpeg_path.join("alvr_build")
+        let primary = base_ffmpeg_path.join("alvr_build");
+        if primary.exists() {
+            primary
+        } else {
+            // As a fallback, assume libavutil is a part of the same
+            // file structure as the rest of ffmpeg:
+            let lib_path = PathBuf::from(
+                pkg_config::probe_library("libavutil")
+                    .unwrap()
+                    .link_paths
+                    .first()
+                    .unwrap(),
+            );
+            lib_path.parent().unwrap().to_path_buf()
+        }
     } else {
-        ffmpeg_path
+        base_ffmpeg_path
     }
 }
 
+#[cfg(target_os = "linux")]
+fn cache_ffmpeg_path() {
+    File::create("../../target/alvr_ffmpeg_path")
```

Comment:
Don't assume the position of the target folder. Instead use `afs::target_dir()`. I'm not on Linux, but on my mac/windows machine I configure cargo to generate the target folder in `/tmp`/`%TEMP`.

Reply:
This is `include_str!`'d on ]the other end](https://github.com/ckiee/ALVR/blob/4ca2f2778f46c5583330449ff745dbcae032cdd4/alvr/launcher/src/commands.rs#L218), so I can't call that runtime function in there without a proc macro. Could move it to `build` but I think that'd be just cluttering that instead (and possibly the distributed zips)

Reply:
if the file is temporary use a temp directory (i think xshell or even std has some helper to resolve the system temp folder).

Reply:
Getting the temp directory on the reading side is also not compile-time. It has to be on a constant path.